### DISCUSSION TITLE
Allow creation of a content type entry from within related content type

### DISCUSSION THREAD

**Discussion by lsliwaradioluz** [strapi/strapi#5882] [Legacy Topics]
I am building a blog using Strapi. I have created two content types - Article and Tag - with many-to-one relation. Right now I have to create Tags separately so that I can use them later on. 

It would be nice if I could create new Tag entries on the fly, while creating a new Article entry - without the neccessity for:

-> saving my current work 
-> leaving the Article editor
-> entering list of Tags
-> adding and saving a new Tag entry
-> going back to my previously saved article to use the refreshed Tags list

The input currently used to filter existing Tags could be used to enter new ones (if they don't exist already). Upon saving my article, my newly created tags would be saved as well. 

I imagine that functionality would only be available for content types with one field at most, like the aforementioned Tag content type that only contains the "name" field. 

![Strapi-tag-create](https://user-images.githubusercontent.com/39009379/79751577-1a795d00-8313-11ea-949d-82028dff15f2.png)

**Comment by derrickmehaffy** [comment]
@soupette thoughts on making this a feature request, else I can convert it to a discussion

**Comment by soupette** [comment]
To be honest it's a really complex topic as we might need another ui to do so (it's not a quick development)...
So maybe converting it to a discussion might be the best idea.

**Reply by derrickmehaffy** [reply]
done :)

**Comment by MattieBelt** [comment]
#5125 Related
